tests/integration/test_npm.py

```diff
@@ -77,6 +77,30 @@
             ),
             id="npm_lockfile3_yarn_registry",
         ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/npm-cachi2-multiple-packages.git",
+                ref="b37466c6a680bb82e087c17ac0232b554f7f2cc6",
+                packages=(
+                    {"path": "first_pkg", "type": "npm"},
+                    {"path": "second_pkg", "type": "npm"},
+                ),
+                check_vendor_checksums=False,
+            ),
+            id="npm_multiple_packages_lockfile1",
+        ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/npm-cachi2-multiple-packages.git",
```

Comment:
Could you pls update the Readme file in the test data repo with the expected behavior info?

Reply:
updated: https://github.com/cachito-testing/npm-cachi2-multiple-packages/blob/lockfile-v1/README.md